src/core/test_data/bootstrapper.aliasconfig

```diff
@@ -0,0 +1,18 @@
+[command]
+cmd = //tools/service_bootstrapper --
+
+[nestedflag "language"]
```

Comment:
This was going to help with recursion if using an Alias instead of AliasConfig as the input to Command, as it would allow both kinds of flags to exist in an Alias - but see your point and have changed it back and used AliasConfig for these instead.